tokio-tls 0.2.1 MSRV raised to 1.28.0 by pkg-config, v0.1.x CI broken

## Version

tokio-tls 0.2.1

## Platform

x86_64-unknown-linux-gnu

## Subcrates

tokio-tls

## Description

MSRV tested and advertised for current tokio-tls 0.2.1 release and current v0.1.x branch is 1.26.0. However [pkg-config 0.3.15 was recently released with MSRV raised to 1.28.0](https://github.com/rust-lang/pkg-config-rs/blob/master/CHANGELOG.md#0315---2019-07-25) (rust-lang/pkg-config-rs#76). Yes, this is an MSRV bump in a PATCH release, but Tokio isn't in much position to complain about that.

Here is the dependency tree:

``` txt
pkg-config v0.3.15
[build-dependencies]
└── openssl-sys v0.9.48
    ├── native-tls v0.2.3
    │   └── tokio-tls v0.2.1 (/home/david/src/tokio/tokio-tls)
    └── openssl v0.10.24
        └── native-tls v0.2.3 (*)
        [dev-dependencies]
        └── tokio-tls v0.2.1 (/home/david/src/tokio/tokio-tls) (*)
```

In v0.1.x CI of MSRV 1.26.0 (via `cargo check --all`) this leads to the following (originally found via #1368): 

https://dev.azure.com/tokio-rs/Tokio/_build/results?buildId=1705

``` txt
  Compiling pkg-config v0.3.15
error[E0658]: use of unstable library feature 'collections_range': might be replaced with `Into<_>` and a type containing two `Bound` values (see issue #30877)
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.15/src/lib.rs:75:23
   |
75 | use std::ops::{Bound, RangeBounds};
   |                       ^^^^^^^^^^^

error[E0658]: use of unstable library feature 'collections_range': might be replaced with `Into<_>` and a type containing two `Bound` values (see issue #30877)
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.15/src/lib.rs:244:12
    |
244 |         R: RangeBounds<&'a str>
    |            ^^^^^^^^^^^^^^^^^^^^

error[E0599]: no method named `start_bound` found for type `R` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.15/src/lib.rs:246:40
    |
246 |         self.min_version = match range.start_bound() {
    |                                        ^^^^^^^^^^^

error[E0599]: no method named `end_bound` found for type `R` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.15/src/lib.rs:251:40
    |
251 |         self.max_version = match range.end_bound() {
    |                                        ^^^^^^^^^

error: aborting due to 4 previous errors
```

--- a/tokio-test/Cargo.toml
+++ b/tokio-test/Cargo.toml
@@ -20,6 +20,7 @@ Testing utilities for Tokio- and futures-based code
 categories = ["asynchronous", "testing"]
 
 [dependencies]
+tokio = { version = "=0.2.0-alpha.1", path = "../tokio" }
 tokio-executor = { version = "=0.2.0-alpha.1", path = "../tokio-executor" }
 tokio-io = { version = "=0.2.0-alpha.1", path = "../tokio-io" }
 tokio-sync = { version = "=0.2.0-alpha.1", path = "../tokio-sync" }
@@ -29,6 +30,4 @@ futures-core-preview = "=0.3.0-alpha.18"
 pin-convert = "0.1.0"
 
 [dev-dependencies]
-tokio = { version = "=0.2.0-alpha.1", path = "../tokio" }
-
 futures-util-preview = "=0.3.0-alpha.18"

--- a/tokio-test/src/lib.rs
+++ b/tokio-test/src/lib.rs
@@ -14,6 +14,18 @@ pub mod io;
 mod macros;
 pub mod task;
 
+/// Runs the provided future, blocking the current thread until the
+/// future completes.
+///
+/// For more information, see the documentation for
+/// [`tokio::runtime::current_thread::Runtime::block_on`][runtime-block-on].
+///
+/// [runtime-block-on]: https://docs.rs/tokio/0.2.0-alpha.1/tokio/runtime/current_thread/struct.Runtime.html#method.block_on
+pub fn block_on<F: std::future::Future>(future: F) -> F::Output {
+    let mut rt = tokio::runtime::current_thread::Runtime::new().unwrap();
+    rt.block_on(future)
+}
+
 /*
 #[doc(hidden)]
 pub mod codegen {

--- a/tokio-test/tests/block_on.rs
+++ b/tokio-test/tests/block_on.rs
@@ -1,0 +1,28 @@
+#![warn(rust_2018_idioms)]
+#![feature(async_await)]
+
+use std::time::{Duration, Instant};
+use tokio_test::block_on;
+use tokio_timer::Delay;
+
+#[test]
+fn async_block() {
+    assert_eq!(4, block_on(async { 4 }));
+}
+
+async fn five() -> u8 {
+    5
+}
+
+#[test]
+fn async_fn() {
+    assert_eq!(5, block_on(five()));
+}
+
+#[test]
+fn delay() {
+    let deadline = Instant::now() + Duration::from_millis(100);
+    let delay = Delay::new(deadline);
+
+    assert_eq!((), block_on(delay));
+}

